snuba/datasets/configuration/generic_metrics/entities/counters_meta.yaml

```diff
@@ -0,0 +1,51 @@
+version: v1
```

Comment:
may be helpful to have a comment in these files explaining what the `meta` tables are for since the name could mean a lot of things